bin/_test-helpers.sh

```diff
@@ -362,7 +362,14 @@ install_version() {
 
     (
         set -x
-        "$linkerd_path" install | kubectl --context="$context" apply -f - 2>&1
+        # TODO: Use a mix of helm override flags and CLI flags and remove this condition
+        # once stable-2.10 is out
```

Comment:
Can you make an issue for this? A TODO is a little easy to forget and I think this is definitely something that will be good to test as soon as the stable is out.